src/parsed-feature-loading.ts

```diff
@@ -90,24 +90,40 @@ const parseScenarioOutlineExampleSteps = (exampleTableRow: any, scenarioSteps: P
             return processedStepText.replace(`<${nextTableColumn}>`, exampleTableRow[nextTableColumn]);
         }, scenarioStep.stepText);
 
-        let stepArgument;
+        let stepArgument: string | {} = '';
 
         if (scenarioStep.stepArgument) {
-            stepArgument = (scenarioStep.stepArgument as any).map((stepArgumentRow: any) => {
-                const modifiedStepAgrumentRow = {...stepArgumentRow};
-
-                Object.keys(exampleTableRow).forEach((nextTableColumn) => {
-                    Object.keys(modifiedStepAgrumentRow).forEach((prop) => {
-                        modifiedStepAgrumentRow[prop] =
-                            modifiedStepAgrumentRow[prop].replace(
-                                `<${nextTableColumn}>`,
-                                exampleTableRow[nextTableColumn],
-                            );
+            if (Array.isArray(scenarioStep.stepArgument)) {
+                stepArgument = (scenarioStep.stepArgument as any).map((stepArgumentRow: any) => {
+                    const modifiedStepArgumentRow = { ...stepArgumentRow };
+
+                    Object.keys(exampleTableRow).forEach((nextTableColumn) => {
+                        Object.keys(modifiedStepArgumentRow).forEach((prop) => {
+                            modifiedStepArgumentRow[prop] =
+                                modifiedStepArgumentRow[prop].replace(
+                                    `<${nextTableColumn}>`,
+                                    exampleTableRow[nextTableColumn],
+                                );
+                        });
                     });
-                });
 
-                return modifiedStepAgrumentRow;
-            });
+                    return modifiedStepArgumentRow;
+                });
+            } else {
+                stepArgument = scenarioStep.stepArgument;
+
+                if (
+                    typeof scenarioStep.stepArgument === 'string' ||
+                    scenarioStep.stepArgument instanceof String
+                ) {
+                    Object.keys(exampleTableRow).forEach((nextTableColumn) => {
+                        stepArgument = (stepArgument as string).replace(
```

Comment:
It's possible that a doc string might have a field specified more than once, but this code will only replace one instance. It would be better to use a regex with /g to ensure each column gets replaced globally.

Reply:
Ah, nice catch that would indeed be quite possible. I'll update the tests to reflect this as well. 

I believe this is a problem also with the current implementation of scenario outlines and placeholders in steps and arguments (titles use regex an works fine). Do you want me to change those into regex checks as well? (I'll just have to come up with some good test scenario where you'd write something like `Given I <placeholder> <placeholder>` :thinking:

Reply:
The fix now uses regex.